data/deep missions.txt

```diff
@@ -339,25 +339,24 @@ mission "Deep: Mystery Cubes 0"
 			
 			branch interested
 				has "deep: interested in cubes"
-			label interested
+			
 			choice
 				`	"Is there anything else that you need me to do?"`
 					goto next
-				`	"What is it that those cubes are for?"`
```

Comment:
You can't use the same label twice, which is why I rearranged it (errors.txt was complaining).
The Kestrel naming mission uses circular labels, so we just need to draw from that instance.
(It also wasn't clear that you wanted to provide both options since you set the condition earlier)